doc/topics.yaml

```diff
@@ -58,6 +58,13 @@
 # The editorial guidelines are intended to evolve as we gain more experience
 # merging/managing topics.
 topics:
+- topic: chart
+  description: Packages that facilitate charting or graphing.
+  aliases:
+  - charts
+  - graph
```

Comment:
I think the plural form (`charts`) is fine here, however, `graph` and `diagram` could mean different things (e.g. directed graph algorithms or flow chart diagrams), and I'd rather keep them separate. Not sure about the `plot`, it seems that `plot` is a subtype of charts, and it would be worth to distinguish between them, if there is a difference.

I think `charts` should stay and the rest could be removed.